react/src/component/DeviceDetailView.jsx

```diff
@@ -158,7 +158,7 @@ export default class DeviceDetailView extends React.Component {
     }
 
     getApplePhoneCase(item) {
-        return <div className='deviceContainer' style={deviceStyle}>
+        return <div className='deviceDisplay' style={deviceStyle}>
```

Comment:
Confusing name of deviceDisplay, could we change it to deviceScreen or sth?

Reply:
fixed